src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -308,6 +305,60 @@ module.exports = class ListObjectsCommandGenerator extends BaseCommandGenerator
 		});
 	}
 
+	_logImportedObjects(importedObjects) {
+		if (importedObjects.length) {
+			NodeUtils.println(TranslationService.getMessage(OUTPUT.IMPORTED_OBJECTS), NodeUtils.COLORS.RESULT);
+			importedObjects.forEach(objectImport => {
+				NodeUtils.println(
+					TranslationService.getMessage(OUTPUT.OBJECT_IMPORTED, objectImport.customObject.type, objectImport.customObject.id),
+					NodeUtils.COLORS.RESULT
+				);
+				this._logReferencedFileImportResult(objectImport.referencedFileImportResult);
+			});
+		}
+	}
+
+	_logUnImportedObjects(unImportedObjects) {
+		if (unImportedObjects.length) {
+			NodeUtils.println(TranslationService.getMessage(OUTPUT.UNIMPORTED_OBJECTS), NodeUtils.COLORS.WARNING);
+			unImportedObjects.forEach(objectImport => {
+				NodeUtils.println(
+					TranslationService.getMessage(
+						OUTPUT.OBJECT_IMPORT_FAILED,
+						objectImport.customObject.type,
+						objectImport.customObject.id,
+						objectImport.customObject.result.message
+					),
+					NodeUtils.COLORS.WARNING
+				);
+			});
+		}
+	}
+
+	_logReferencedFileImportResult(referencedFileImportResult) {
+		const importedFiles = referencedFileImportResult.successfulImports;
+		const unImportedFiles = referencedFileImportResult.failedImports;
+
+		if (importedFiles.length || unImportedFiles.length) {
```

Comment:
Agree with the check. However, I implemented a but different to simplify I think, I instead did:
```if (Array.isArray(importedFiles) && Array.isArray(unImportedFiles)) {
			return;
		}
```
This is because we need to show a message if either importedFiles or unImportedFiles is not empty.

Reply:
Not sure if that's the same. What if there are none unImportedFiles. Maybe we get a null istead of an empty array. That would mean that the following block of code would not run.

Reply:
Change made.